pkg/runtime/path.go

```diff
@@ -0,0 +1,75 @@
+package runtime
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/alibaba/pouch/apis/types"
+)
+
+var (
+	runtimeDir                    = "runtimes"
+	runtimeDirPerm    os.FileMode = 0700
+	runtimeScriptPerm os.FileMode = 0700
+
+	runtimePaths map[string]string
+)
+
+// InitialPath creates runtime script for those has args
+func InitialPath(baseDir string, runtimes map[string]types.Runtime) error {
+	// initializes map keep runtime script path
+	if runtimePaths == nil {
+		runtimePaths = make(map[string]string)
+	}
+
+	dir := filepath.Join(baseDir, runtimeDir)
+
+	// remove runtime scripts last generated, since runtime config may changed
+	// every time daemon start.
+	if err := os.RemoveAll(dir); err != nil {
+		return fmt.Errorf("failed to clean runtime scripts directory %s: %s", dir, err)
+	}
+
+	if err := os.MkdirAll(dir, runtimeDirPerm); err != nil {
+		return fmt.Errorf("failed to new runtime scripts directory %s: %s", dir, err)
+	}
+
+	// create script for runtime who has args
+	for name, r := range runtimes {
+		if len(r.RuntimeArgs) == 0 {
+			if r.Path != "" {
+				runtimePaths[name] = r.Path
+			}
+			continue
+		}
+
+		script := filepath.Join(dir, name)
+		dataSlice := []string{name}
+		if r.Path != "" {
+			dataSlice = []string{r.Path}
+		}
+		for _, arg := range r.RuntimeArgs {
+			dataSlice = append(dataSlice, arg)
+		}
+
+		if err := ioutil.WriteFile(script, []byte(strings.Join(dataSlice, " ")), runtimeScriptPerm); err != nil {
```

Comment:
All of the parameters should be passed to the runtime. Could you please add `$@` to the script？